CMakeLists.txt

```diff
@@ -421,12 +421,6 @@ configure_file("${pcl_config_h_in}" "${pcl_config_h}")
 PCL_ADD_INCLUDES(common "" "${pcl_config_h}")
 include_directories("${CMAKE_CURRENT_BINARY_DIR}/include")
 
-### ---[ Set up for tests
-enable_testing()
```

Comment:
I deleted this because it is alreadly being invoked in the right place
https://github.com/PointCloudLibrary/pcl/blob/72f41b60a539cd1da67d1329b57222290122a0bb/test/CMakeLists.txt#L20